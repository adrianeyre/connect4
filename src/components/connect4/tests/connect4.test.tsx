import React from 'react';
import { shallow } from 'enzyme';

import Connect4 from '../connect4';
import IConnect4Props from '../interfaces/connect4-props';

describe('Connect 4', () => {
	it('Should render correctly', () => {
		const defaultProps: IConnect4Props = {};
		const connect4 = shallow(<Connect4 {...defaultProps} />);
		expect(connect4).toMatchSnapshot();
	});
});