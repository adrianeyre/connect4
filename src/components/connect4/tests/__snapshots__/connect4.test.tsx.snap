// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect 4 Should render correctly 1`] = `ShallowWrapper {}`;
